internal/rpc/user_devices.go

```diff
@@ -430,6 +430,16 @@ func (s *userDeviceRPCServer) deviceModelToAPI(ud *models.UserDevice) *pb.UserDe
 				Expiration: timestamppb.New(vc.ExpirationDate),
 			}
 		}
+
+		if sd := vnft.R.VehicleTokenSyntheticDevice; sd != nil {
```

Comment:
Are we loading this relation? Have we tested this?

Reply:
Like, I think you're on the hook for checking all the callers of `deviceModelToAPI` and seeing if they load the relation you want. Is this good? Probably not.